Bottom: 9676d2bd39376bf42d61269517fc6cb07c23b6ef
Top:    92ae60f4eea5493d51488664a9ab901829b2ff04
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-19 17:14:53 +0000

Refresh of play-with-subcompositing-using

---

diff --git a/src/polygon.c b/src/polygon.c
index 5e79ee8..e16fd55 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -253,9 +253,6 @@ original_poly (PolygonType * p)
       }
   }
   END_LOOP;
-  if (contour == NULL) {
-    printf ("WTF? Got contour==NULL in original_poly\n");
-  }
   poly_PreContour (contour, TRUE);
   /* make sure it is a positive contour */
   if ((contour->Flags.orient) != PLF_DIR)
